### PR TITLE
Create useLabel function and make use of it

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx [hunk 1]
```diff
@@ -23,10 +23,26 @@ export interface PicturesBottomPanelProps extends ResizableBottomPanelProps {
...
 }
 
+/**
+ * Create a memoized localized string for the given key.
+ *
+ * __Note__: This just uses {@link React.useMemo} on the result from {@link i18n}.
+ * @param key The i18n key for the label.
+ * @returns A memoized localized string.
+ */
+function useLabel(key: string) {
```
Comment (toddsouthenbentley) [inline cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx:42]: I feel a better name would be: `useLocalizedString`

Comment (tcobbs-bentley) [reply cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx:42]: I kind of liked the succinctness of useLabel. However, useLocalizedString is more descriptive, so I made the change.

Comment (toddsouthenbentley) [reply cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx:42]: This instance of `useLabel` hasn't been renamed.

Comment (tcobbs-bentley) [reply cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx]: Thanks. I renamed both functions, but VS Code auto-saved all the files when I renamed the other function, but didn't auto-save the one file when I renamed this function (and I didn't notice).

**Review by toddsouthenbentley — APPROVED** [review]

**Pull request merged**